samples/nrf9160/aws_fota/Kconfig

```
@@ -44,11 +44,11 @@ config USE_PROVISIONED_CERTIFICATES
 	  should provide a valid server certificate in `certificates.h` file.
 
 config MQTT_MESSAGE_BUFFER_SIZE
-	int ""
+	int "MQTT message buffer size"
 	default 128
 
 config MQTT_PAYLOAD_BUFFER_SIZE
-	int ""
+	int "MQTT read buffer size"
```

The prompt does not match the define name.

```suggestion
	int "MQTT payload buffer size"
```

same below.

---

Thought I would give it a more intuitive name in the prompt but I'll change it back.

---

Done.